fe/gulpfile.coffee

```diff
@@ -0,0 +1,67 @@
+#!/usr/bin/env coffee
+gulp = require 'gulp'
+jade = require 'gulp-jade'
+copy = require 'gulp-copy'
+styl = require 'gulp-stylus'
+coffee = require 'gulp-coffee'
+concat = require 'gulp-concat'
+runSequence = require 'run-sequence'
+del = require 'del'
+
+jadeopts = {}
+coffeeopts = {}
+stylopts = {}
+
+coffeesrc =
+  'bustracker.js': [
+    'src/main/coffee/channels.coffee'
+    'src/main/coffee/bustracker.coffee'
+  ]
+  'backdoor.js': [
+    'src/main/coffee/channels.coffee'
+    'src/main/coffee/backdoor.coffee'
+  ]
+
+gulp.task 'clean-js', (cb) ->
+  del ['../public/**/*.js'], force: true, cb
+
+gulp.task 'clean-css', (cb) ->
+  del ['../public/**/*.css'], force: true, cb
+
+gulp.task 'clean-html', (cb) ->
+  del ['../public/**/*.html'], force: true, cb
```

Comment:
force: true is only necessary here because it's not a child folder. Alternatively, all the contents of the fe folder could be moved up a notch and then we wouldn't need this